tests/TestCase/Datasource/ModelAwareTraitTest.php

```diff
@@ -32,10 +32,10 @@ class ModelAwareTraitTest extends TestCase
     public function testSetModelClass()
     {
         $stub = new Stub();
-        $this->assertNull($stub->modelClass);
+        $this->assertNull($stub->getProp('modelClass'));
 
         $stub->setProps('StubArticles');
-        $this->assertEquals('StubArticles', $stub->modelClass);
+        $this->assertEquals('StubArticles', $stub->getProp('modelClass'));
```

Comment:
You could use `assertAttributeEquals` to avoid having to build your own property reader.

Reply:
Done

Reply:
Shouldnt we using Same here? This is safer and will be used more common with scalar comparison.